src/Microsoft.DotNet.MSBuildSdkResolver/StringExtensions.cs

```diff
@@ -0,0 +1,15 @@
+namespace Microsoft.DotNet.MSBuildSdkResolver
+{
+    internal static class StringExtensions
+    {
+        public static int FindFirstNotOf(this string s, string chars, int startIndex)
```

Comment:
I'm not a fan of extension methods on primitives, but apparently I'm alone on that on this team.